core/main.mk

```diff
@@ -136,10 +136,12 @@ $(warning ************************************************************)
 $(error Directory names containing spaces not supported)
 endif
 
-ifndef BUILD_EMULATOR
```

Comment:
I suggest to keep this condition and set

BUILD_EMULATOR ?= true

in front of it. That is the least change from the upstream repository and will allow us to experiment easily with prebuilt emulator binaries.